lib/service/service.go

```diff
@@ -2103,22 +2103,21 @@ func (process *TeleportProcess) initSSH() error {
 		trace.Component: teleport.Component(teleport.ComponentNode, process.id),
 	})
 
-	var agentPool *reversetunnel.AgentPool
 	proxyGetter := reversetunnel.NewConnectedProxyGetter()
 
-	var conn *Connector
-	var ebpf bpf.BPF
-	var rm restricted.Manager
-	var s *regular.Server
-	var asyncEmitter *events.AsyncEmitter
-
 	process.RegisterCriticalFunc("ssh.node", func() error {
-		var err error
-		conn, err = process.waitForConnector(SSHIdentityEvent, log)
+		conn, err := process.waitForConnector(SSHIdentityEvent, log)
 		if conn == nil {
 			return trace.Wrap(err)
 		}
 
+		ok := false
```

Comment:
What do you think of using a named `err` return value and checking it for `nil`? Too easy to screw up?

Reply:
I think so. Easy to accidentally shadow.